--- origsrc/tasque-0.1.6/src/Tasque.exe.config.in	2008-06-06 11:15:01.000000000 -0500
+++ src/tasque-0.1.6/src/Tasque.exe.config.in	2008-06-24 10:22:33.093750000 -0500
@@ -1,4 +1,4 @@
 <configuration>
-  <dllmap dll="libX11" target="libX11.so.6" />
-  <dllmap dll="gdk-x11-2.0" target="libgdk-x11-2.0.so.0" />
+  <dllmap dll="libX11" target="cygX11-6.dll" />
+  <dllmap dll="gdk-x11-2.0" target="cyggdk-x11-2.0-0.dll" />
 </configuration>
--- origsrc/tasque-0.1.6/src/Utilities.cs	2008-06-06 11:15:01.000000000 -0500
+++ src/tasque-0.1.6/src/Utilities.cs	2008-06-24 10:33:20.921875000 -0500
@@ -48,11 +48,14 @@
 
 		public static void SetProcessName(string name)
 		{
-			// 15 = PR_SET_NAME
-			if(prctl(15, Encoding.ASCII.GetBytes(name + "\0"), IntPtr.Zero, IntPtr.Zero, IntPtr.Zero) != 0)
+			try
 			{
-				throw new ApplicationException("Error setting process name: " + Mono.Unix.Native.Stdlib.GetLastError());
-			}
+				// 15 = PR_SET_NAME
+				if(prctl(15, Encoding.ASCII.GetBytes(name + "\0"), IntPtr.Zero, IntPtr.Zero, IntPtr.Zero) != 0)
+				{
+					throw new ApplicationException("Error setting process name: " + Mono.Unix.Native.Stdlib.GetLastError());
+				}
+			} catch (EntryPointNotFoundException) {}
 		}
 
 		public static string ReplaceString (string originalString, string searchString, string replaceString)
